integrationTests/relayers/slowTests/framework/testSetup.go

```diff
@@ -164,19 +178,21 @@ func (setup *TestSetup) IssueAndConfigureTokens(tokens ...TestTokenParams) {
 		setup.EthereumHandler.IssueAndWhitelistToken(setup.Ctx, token.IssueTokenParams)
 		setup.MultiversxHandler.IssueAndWhitelistToken(setup.Ctx, token.IssueTokenParams)
 
+		setup.ChainSimulator.GenerateBlocks(setup.Ctx, 10)
+
 		esdtBalanceForSafe := setup.MultiversxHandler.GetESDTChainSpecificTokenBalance(setup.Ctx, setup.MultiversxHandler.SafeAddress, token.AbstractTokenIdentifier)
+		AliceMvxBalance := setup.MultiversxHandler.GetESDTUniversalTokenBalance(setup.Ctx, setup.AliceKeys.MvxAddress, token.AbstractTokenIdentifier)
 		AliceEthBalance := setup.EthereumHandler.GetBalance(setup.AliceKeys.EthAddress, token.AbstractTokenIdentifier)
 
 		setup.mutBalances.Lock()
 		setup.esdtBalanceForSafe[token.AbstractTokenIdentifier] = esdtBalanceForSafe
 
-		//setup.ethBalances[setup.AliceKeys.EthAddress.String()] = make(map[string]*big.Int)
-		setup.ethBalances[setup.AliceKeys.EthAddress.String()][token.AbstractTokenIdentifier] = AliceEthBalance
+		setup.mvxBalances[setup.AliceKeys.MvxAddress.String()][token.AbstractTokenIdentifier] = AliceMvxBalance
+		setup.mvxBalances[setup.BobKeys.MvxAddress.String()][token.AbstractTokenIdentifier] = big.NewInt(0)
```

Comment:
these might be prone to errors if we make the assumption that Bob & Charlie will start the test in all cases with a 0 value. Why not make a function that queries the balance of a provided key and saves it in the mvxBalances map? And invoke it for all three keys: Alice, Bob, Charlie